R/qenv-concat.R

```diff
@@ -28,6 +28,7 @@ setGeneric("concat", function(x, y) standardGeneric("concat"))
 #' @export
 setMethod("concat", signature = c("qenv", "qenv"), function(x, y) {
   y@id <- c(x@id, y@id)
+  y@code_dependency <- c(x@code_dependency, y@code_dependency)
   y@code <- c(x@code, y@code)
   y@warnings <- c(x@warnings, y@warnings)
   y@messages <- c(x@messages, y@messages)
```

Comment:
```suggestion
  lapply(setdiff(methods::slotNames(x), "env"), function(slot) {
    do.call(`<<-`, list(str2lang(sprintf("y@%s", slot)), c(do.call("@", list(x, slot)), do.call("@", list(y, slot)))))
  })
```

Reply:
looks cool, but the readability is really unpleasant for this change. I would rather not include that

Reply:
Yeah, fair enough.